src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getS() {
+        return 3.14 * Math.pow(this.radius, 2);

```

Comment:
Consider using `Math.PI` instead of `3.14` for better precision in calculating the area of the circle.